resolutionJETS/resolutionJETStree.C

```diff
@@ -2,463 +2,376 @@
 #include "../common/binningheader.h"
 #define NELEMS(arr) (sizeof(arr)/sizeof(arr[0]))
 
+#include <TROOT.h>
+#include <TH1F.h>
+#include <TSystem.h>
+#include <TFile.h>
+#include <TH2F.h>
+#include <TCanvas.h>
+#include <TH2.h>
+#include <THStack.h>
+#include <TLegend.h>
+
+#include <iostream>
+
+const int njettypes = 3;
+const int firstEtaBin = 10;
+const int nInputs = 1;
+
+struct plottingStyleData
+{
+  Color_t color_jets[njettypes] = {kOrange+2, kMagenta+2, kRed+2}; //, kGreen+2, kBlue+2};
+  int marker_jets[njettypes] = {21, 20, 22};//, 29, 47};
+  int linestyle_jets[njettypes] = {1, 2, 4};//, 8, 9};
+  TString str_jet_type[njettypes] = {"track", "calo", "all"};
+  TString str_jet_type_plot[njettypes] = {"Charged Jets (Track)", "Calo Jets (FHCAL+FEMC)", "All Jets"};
+  TString collisionSystem = "Pythia 6, e+p: 18#times#kern[0.2]{100} GeV^{2} High Q^{2}";
+  TString format;
+};
+
+void plotResoOrScale(TH1F *scaleData[nInputs][njettypes][16], TString outputDir, plottingStyleData style, float yMin, float yMax, TString xLabel, TString yLabel);
+void plotSpectra(TH2F *spectra[nInputs][njettypes], plottingStyleData style, TString title, TString outputFormat, TH1F *reco[nInputs][njettypes]=nullptr, TH1F *truth[nInputs][njettypes]=nullptr);
+void plotEfficiency(TH1F *h_matched_count[nInputs][njettypes], TH1F *h_truth_count[nInputs][njettypes], plottingStyleData style, TString outputDir);
+
+
 void resolutionJETStree(
-    TString suffix            = "pdf"
+    TString suffix            = "png"
```

Comment:
Please change back to PDF. We tend to stick with vector graphics when we can

Reply:
Fixed with https://github.com/tlprotzman/AnalysisSoftwareEIC/commit/f9f79492e8b9fce5f7c267b65c81a357f7c395f8